contracts/Sales.sol

```diff
@@ -261,14 +261,14 @@ contract Sales is DSMath { // Auctions
 		require(hasSecs(sale));
 		require(sha256(abi.encodePacked(sechs[sale].secD)) == sechs[sale].sechD);
         sales[sale].taken = true;
-        if (sales[sale].bid > (loans.dedu(sales[sale].loani))) {
-            require(tokes[sale].transfer(sales[sale].lend, loans.lentb(sales[sale].loani)));
+        if (sales[sale].bid > (loans.dedub(sales[sale].loani))) {
+            require(tokes[sale].transfer(sales[sale].lend, loans.lent(sales[sale].loani)));
             if (agent(sale) != address(0)) {
                 require(tokes[sale].transfer(sales[sale].agent, loans.lfee(sales[sale].loani)));
             }
             require(tokes[sale].approve(address(med), loans.lpen(sales[sale].loani)));
             med.push(loans.lpen(sales[sale].loani), tokes[sale]);
-            require(tokes[sale].transfer(sales[sale].bor, add(sub(sales[sale].bid, loans.dedub(sales[sale].loani)), loans.back(sales[sale].loani))));
+            require(tokes[sale].transfer(sales[sale].bor, sub(add(sales[sale].bid, loans.back(sales[sale].loani)), loans.dedu(sales[sale].loani))));
```

Comment:
transfer ((bid + repaid) - (principal + interest + agent fee + liquidation penalty)) to borrower